sys-fs/mergerfs/mergerfs-2.24.2.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs autotools
+
+DESCRIPTION="A featureful FUSE union filesystem across multiple drives"
+HOMEPAGE="https://github.com/trapexit/mergerfs"
+# tarball under download/ has prebuilt ChangeLog and AUTHORS,
+# and more importantly version.hpp,
+# so use it instead of git tag tarball
+SRC_URI="https://github.com/trapexit/mergerfs/releases/download/${PV}/${P}.tar.gz"
+
+LICENSE="ISC"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+xattr"
+# mergerfs bundles its own _patched_ copy of libfuse.
+# No easy way around it yet. :(
+#IUSE="${IUSE} +external-fuse"
+
+DEPEND="xattr? ( sys-apps/attr )"
+RDEPEND="${DEPEND}"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-Makefile-don-t-touch-LDFLAGS.patch
+)
+
+DOCS=(
+	AUTHORS
+	ChangeLog
+	README.md
+)
+
+domake() {
+	local makeargs=( "$@" )
+	local makevars=(
+		GIT=''
+		MKDIR="$(type -P mkdir)"
+		TOUCH="$(type -P touch)"
+		CP="$(type -P cp)"
+		RM="$(type -P rm)"
+		LN="$(type -P ln)"
+		INSTALL="$(type -P install)"
+		STRIP='true'
+		PANDOC='' # don't rebuild manpages
+
+		XATTR_AVAILABLE="$(usex xattr 1 0)"
+		PREFIX="${EPREFIX:-/usr}"
+		CXX="$(tc-getCXX)"
+		OPTS="${CPPFLAGS} ${CXXFLAGS}"
+	)
+
+	emake "${makevars[@]}" "${makeargs[@]}"
+}
+
+src_prepare() {
+	default
+
+	# don't rebuild version.hpp
+	cat > tools/update-version <<EOFF
```

Comment:
`|| die`

Also you can use `<<-EOF` to strip whitespace and allow indenting stuff, e.g.:

```
  cat > foo <<-EOF || die
    bar baz
  EOF

Reply:
Oh, good catch!
Forgot about `|| die`.
(so used to `set -e`)

Will convert to the indented here-doc too.